app/src/sharedTest/java/org/oppia/android/app/parser/ListItemLeadingMarginSpanTest.kt

```diff
@@ -711,6 +827,128 @@ class ListItemLeadingMarginSpanTest {
     assertThat(shadowCanvas.getDrawnTextEvent(3).text).isEqualTo("٢.")
   }
 
+  // TODO(#3840): Make this test work on Espresso & Robolectric.
```

Comment:
Should just be 'Espresso' since it's already configured to work on Robolectric.

Reply:
Done.